components/kyma-environment-broker/cmd/broker/update_test.go

```diff
@@ -1755,6 +1858,7 @@ func TestUpdateSCMigrationSuccess(t *testing.T) {
 	i, err = suite.db.Instances().GetByID(id)
 	assert.NoError(t, err, "getting instance after update")
 	assert.True(t, i.InstanceDetails.SCMigrationTriggered, "instance SCMigrationTriggered after update")
+	time.Sleep(5 * time.Millisecond)
```

Comment:
Sometimes the test fails, I'm waiting for removing the migration stuff so the test will be also removed